themes/OpenFisca-2022/layouts/_default/styleguide.html

```diff
@@ -21,7 +21,7 @@
 
     {{ $testimonials := slice }}
     {{ $testimonials = $testimonials | append (dict "title" "“Better policy outcomes and enhanced service delivery”" "image" "oecd.png") }}
-    {{ $testimonials = $testimonials | append (dict "title" "Digital Public Good nominee" "image" "dpg.png") }}
+    {{ $testimonials = $testimonials | append (dict "title" "Edge of Government Award 2023" "image" "wgs.png") }}
```

Comment:
If that is the `alt`, it should be `World Government Summit` as this is what is written on the image 🙂

Reply:
This is the `title`, the `alt` could be added by passing `author` parameter.

Reply:
But your comment made me realize that it was easily possible to add rendering conditions to avoid empty attributes and tags, which I did.